.github/workflows/deployDev3.yml

```diff
@@ -0,0 +1,123 @@
+name: Deploy Dev2
```

Comment:
I think this needs to be Dev2

Reply:
It absolutely does! Thanks for the catch. Fixed!